charts/k8s-infra/values.yaml

```diff
@@ -212,6 +212,10 @@ presets:
     # DO NOT CHANGE BELOW TO false, causes data duplication in case attributes mismatched.
     override: true
     envResourceAttributes: ""
+  k8sEvents:
+    enabled: true
+    authType: serviceAccount
+    namespaces: []
```

Comment:
Do include a comment that if no namespace is passed, it defaults to all namespace.